src/app/basic_examples/hashing/page.mdx

```diff
@@ -0,0 +1,224 @@
+export const metadata = {
+  title: "Hasing with keccak256 • Stylus by Example",
+  description:
+    "A simple solidity ABI encode and decode example",
+};
+
+{/* Begin Content */}
+
+
+## Hashing with keccak256
+
+keccak256 computes the Keccak-256 hash of the input.
+
+Some use cases are:
+ - Creating a deterministic unique ID from a input
+ - Commit-Reveal scheme
+ - Compact cryptographic signature (by signing the hash instead of a larger input)
+
```

Comment:
There's an internal method to calculate the keccak hash in the `crypto` module (it uses the keccak256 function from the allow packages), in case we want to show that in this page: https://github.com/OffchainLabs/stylus-sdk-rs/blob/main/stylus-sdk/src/crypto.rs#L22

But if we're using the one in the sha3 crate, we should probably mention that keccak256 is done through the sha3 library, and a link to it.